### PR TITLE
changes for dropdown as per requirement

### PR DESCRIPTION
**Pull request by navichawla92**

### CODE REVIEW

File: src/components/search-columns.js [hunk 1]
```diff
@@ -17,6 +19,46 @@ function renderCheckbox(column, query, onCheckChange) {
   ) : (
     ""
```
Comment (tvarsis) [inline src/components/search-columns.js:20]: Remove commented code

File: src/components/search-columns.js [hunk 2]
```diff
@@ -97,13 +139,20 @@ const SearchColumns = ({ columns, query, onChange }) => {
...
     onChange({
       ...query,
-      [event.target.name]: query[event.target.name] === false && event.target.checked ? undefined : event.target.checked
+      [name]: value
     });
   };
 
+ /* const onCheckChangeOld = (event) => {
+    onChange({
+      ...query,
```
Comment (tvarsis) [inline src/components/search-columns.js:151]: Remove commented code

File: src/components/search-columns.js [hunk 3]
```diff
@@ -17,6 +19,46 @@ function renderCheckbox(column, query, onCheckChange) {
...
+      style={{ width: '100%' }}
+      name={column.property}
+      placeholder={column.filterPlaceholder || ""}
+      defaultValue={query[column.property] || "all"}
+      onChange={(value) => onCheckChange(column.property, value)}
+    >
+      <Option value={'all'}>ALL</Option>
+      <Option value={true}>True</Option>
+      <Option value={false}>False</Option>
+      <Option value={null}>NULL</Option>
```
Comment (tvarsis) [inline src/components/search-columns.js:36]: Lets change the naming of these to:

All
True
False
Undefined

To be more clear and follow better casings.

**Review by tvarsis — CHANGES_REQUESTED** [review]
Please fix the three comments, and then I will merge this. Thanks!

File: src/components/search-columns.js (hunk 3 above)
Comment (navichawla92) [reply src/components/search-columns.js:36]: @tvarsis I will remove the commented code but for this part.

**Pull request closed without merging**